upgrade.md

```diff
@@ -17,6 +17,139 @@ The `storagePath` method of the `Illuminate\Contracts\Foundation\Application` in
 
     public function storagePath($path = '');
 
+<a name="mail"></a>
+### Mail
+
+<a name="symfony-mailer"></a>
+#### Symfony Mailer
+
+**Likelihood Of Impact: High**
+
+Swift Mailer is discontinued as of December 2021 and was therefore replaced with Symfony Mailer 6.0 (See https://symfony.com/blog/the-end-of-swiftmailer)
+Because of this SwiftMailer plugins and transports will no longer work with the new implementation.
+
+##### Updated Content Methods
+`send`, `html`, `text` & `plain` don't return the number of recipients anymore but an instance of `Illuminate\Mail\SentMessage` which contains the `Symfony\Component\Mailer\SentMessage` instance
+
+##### Renamed Swift methods to Symfony
+
+All Swift methods were renamed to an equivalent with Symfony in the method name. For example:
+
+    // BEFORE
+    $this->withSwiftMessage(function ($message) {
+        $message->getHeaders()->addTextHeader(
+            'Custom-Header', 'Header Value'
+        );
+    });
+
+    // AFTER
+    use Symfony\Component\Mime\Email;
+
+    $this->withSymfonyMessage(function (Email $message) {
+        $message->getHeaders()->addTextHeader(
+            'Custom-Header', 'Header Value'
+        );
+    });
+
+Please also take a look at the [Symfony Mailer Docs](https://symfony.com/doc/6.0/mailer.html#creating-sending-messages) for all possible interactions with the Email message.
+
+Here is a list of all Methods which need to be renamed if they are used:
+    
+    Message::getSwiftMessage();
+    Message::getSymfonyMessage();
+
+    Mailable::withSwiftMessage($callback);
+    Mailable::withSymfonyMessage($callback);
+
+    MailMessage::withSwiftMessage($callback);
+    MailMessage::withSymfonyMessage($callback);
+
+    Mailer::getSwiftMailer();
+    Mailer::getSymfonyTransport();
+
+    Mailer::setSwiftMailer($swift);
+    Mailer::setSymfonyTransport(TransportInterface $transport);
+    
+    MailManager::createTransport($config);
+    MailManager::createSymfonyTransport($config);
+
+##### `Illuminate\Mail\Message`
+The Illuminate\Mail\Message class now contains an instance of Symfony\Component\Mime\Email instead of Swift_Message so all forwarding calls in userland will need to be updated. Here are some examples which were possible with SwiftMailer but are not possible anymore with Symfony Mailer (most people are probably using the laravel style already):
```

Comment:
```suggestion
##### `Illuminate\Mail\Message`

The `Illuminate\Mail\Message` class now contains an instance of `Symfony\Component\Mime\Email` instead of `Swift_Message` so all forwarding calls in user-land will need to be updated. Here are some examples which were possible with SwiftMailer but are not possible anymore with Symfony Mailer (most people are probably using the Laravel style already):
```